app/scripts/modules/core/src/pipeline/config/triggers/cron/CronTrigger.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import Select, { Option } from 'react-select';
+
+import { CronMinutes } from './CronMinutes';
+import { CronHourly } from './CronHourly';
+import { CronDaily } from './CronDaily';
+import { CronWeekly } from './CronWeekly';
+import { CronMonthly } from './CronMonthly';
+import { CronAdvance } from './CronAdvance';
+import { ICronTriggerConfigProps } from './cronConfig';
+import { BaseTrigger } from 'core/pipeline';
+
+import './cronTrigger.less';
+
+export interface ICronTriggerState {
+  activeTab: string;
+}
+
+export class CronTrigger extends React.Component<ICronTriggerConfigProps, ICronTriggerState> {
+  private tabOptions = [
+    { label: 'minutes', value: 'minutes', regex: [/^(0 0\/\d+ \* 1\/1 \* \? \*)$/g] },
+    { label: 'hourly', value: 'hourly', regex: [/^(0 \d+ 0\/\d+ 1\/1 \* \? \*)$/g] },
+    { label: 'daily', value: 'daily', regex: [/^(0 0 0 1\/\d+ \* \? \*)$/g, /^(0 \d+ \d+ \? \* MON\-FRI \*)$/g] },
```

Comment:
This expression should be matching for `daily`, but it is not:
```
0 0 20 1/1 * ? *
```
I'm just starting to run through some of our existing triggers, will update this thread with others I find...

Reply:
good catch. Fixed the regex.